daemon-tests/tests/rollover.rs

```diff
@@ -190,42 +182,110 @@ async fn maker_accepts_rollover_after_commit_finality() {
 /// The first rollover is done with `example_1`.
 /// The second rollover is done with `example_0` (we re-use it)
 #[otel_test]
-async fn retry_rollover_an_open_cfd() {
-    let contract_setup_oracle_data = OliviaData::example_0();
-    let contract_setup_oracle_data_announcement = contract_setup_oracle_data.announcement();
+async fn retry_rollover_an_open_cfd_from_contract_setup() {
```

Comment:
🙃 What about naming this `given_contract_setup_completed_when_taker_fails_first_rollover_can_retry`?